index.html

```diff
@@ -10,17 +10,29 @@
     <div class="mobile-container">
         <header class="top">
         <a class="link-name" href="#">Shuhad Loofer</a>
-        <nav class="nav">
-            <a><img src="./images/Menu.svg" alt="Menu"/></a>
-        </nav>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
         <nav class="nav1">
-            <a href="#">Profile</a>
-            <a href="#">About</a>
-            <a href="#">Contact</a>
-            <a href="#"><img src="./images/Union.svg" alt="Email"/></a>
+            <ul class="nav-menu" >
+                <li class="nav-item">
+                    <a class="nav-link" href="#Profile">Profile</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#About">About</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#Contact">Contact</a>
+                </li>
+                <li class="nav-email" class="nav-item">
+                    <a href="#"><img src="./images/Union.svg" alt="Email"/></a>
+                </li>
+            </ul>
```

Comment:
- [ ] As requested by previous reviewer, consider making sure all the nav-lists links(Portfolio, Contact, about,...) and other sections are still visible on all Desktop screen widths range.

> * [ ]  Kindly make sure all the **nav-lists links(Portfolio, Contact, about,...)** and **other sections** are still visible on all Desktop screen widths range as they disappear at a certain range of screen widths. I suggest you take a look at the Gif below and fix the issue to make your project professional.
>   ![nav-issues](https://user-images.githubusercontent.com/83042742/173393751-df322848-64f0-4451-bdb3-cd333bc76c00.gif)
>